scripts/release.cjs

```diff
@@ -1,7 +1,28 @@
 const fs = require('fs');
+const glob = require('glob');
 const path = require('path');
 const execa = require('execa');
 
+const root = path.resolve(__dirname, '..');
+
+/** Update  */
+function updateLocalDepVersions(dependencyName, newVersion) {
+  const pkgJsonLocs = glob.sync('**/*/package.json', {
```

Comment:
There may be a smarter way to do this, but I didn’t want to lock us further into some Lerna feature (possibly with consequences) 🤷🏻

Reply:
I would have tackled this the same way, fwiw

Reply:
yea, the less we rely on Lerna the better. +1 for doing it this way as well!

The one concern is that yarn needs to update the lockfile after this happens, and that we get into a place where every `yarn install` creates a new diff on main until committed. Maybe add a `yarn install` run after this fn is run? Or, just confirm that that's not needed.